files/build_templates/docker_image_ctl.j2

```diff
@@ -238,7 +242,9 @@ start() {
             redis_dir=`echo $redis_dir_list | cut -d " " -f $id`
             REDIS_MNT=" -v $redis_dir:$redis_dir:rw "
         fi
-
+        # Containers in the docker network will use the syslog-driver as log-driver. This is mainly applicable for Multi NPU system
+        LOG_OPTS="--log-driver=syslog --log-opt syslog-address=udp://127.0.0.1:514 --log-opt syslog-format=rfc5424 \
```

Comment:
Should there be a check for docker_image_run_opt has --log-driver=json-file ?

Reply:
@SuvarnaMeenakshi syslogs will not work if the log-driver is json on containers running the docker bridge network, so I think we can ignore --log-driver=json-file  in this case.

Reply:
Could you giver proof or reference that "syslogs will not work if the log-driver is json on containers running the docker bridge network"?

Reply:
Fixed in the latest commit